pylib/anki/stats.py

```diff
@@ -746,9 +746,9 @@ def _eases(self) -> Any:
                 "id > %d" % ((self.col.sched.day_cutoff - (days * 86400)) * 1000)
             )
         if lims:
-            lim = "where " + " and ".join(lims)
+            lim = "where type != {REVLOG_RESCHED} and " + " and ".join(lims)
```

Comment:
These should be f-strings:
```suggestion
            lim = f"where type != {REVLOG_RESCHED} and " + " and ".join(lims)
```